Microsoft.NET.Build.Containers/CreateNewImage.cs

```diff
@@ -205,6 +205,11 @@ public override bool Execute()
             return !Log.HasLoggedErrors;
         }
 
+        if (ImageTags.Length < 1) {
+            Log.LogError($"No tags were passed in the argument {nameof(ImageTags)}");
+            return !Log.HasLoggedErrors;
+        }
```

Comment:
We could also let MSBuild validate this by marking `ImageTags` as `[Required]`. Thoughts on preferred experience?

Reply:
If MSBuild sees an empty property being passed into the [Required] one and reports the same error, I'd lean towards that personally. Won't block on it though

Reply:
It's not the _same_ error (it's an MSB code) but it's functionally identical. Switched to that.